https://bugs.kde.org/show_bug.cgi?id=440372
Regressed by https://github.com/KhronosGroup/EGL-Registry/commit/4e5d0cb9d025

src/plugins/platforms/x11/common/eglonxbackend.cpp:227:66: error: cannot initialize a parameter of type 'EGLNativeWindowType' (aka 'void *') with an lvalue of type 'xcb_window_t' (aka 'unsigned int')
        surface = eglCreateWindowSurface(eglDisplay(), config(), window, nullptr);
                                                                 ^~~~~~

--- src/plugins/platforms/x11/common/eglonxbackend.cpp.orig	2021-08-31 11:41:03 UTC
+++ src/plugins/platforms/x11/common/eglonxbackend.cpp
@@ -6,6 +6,8 @@
 
     SPDX-License-Identifier: GPL-2.0-or-later
 */
+#undef EGL_NO_PLATFORM_SPECIFIC_TYPES
+#define USE_X11
 #include "eglonxbackend.h"
 // kwin
 #include "main.h"
